.github/workflows/integration-tests.yaml

```diff
@@ -39,11 +39,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Build All Variants
-        run: mvn clean install -Pprod -Pjpa -Pinfinispan -Pkafka -Pstreams -Pasyncmem -DskipTests -pl !tests
-      - name: Run Integration Tests - kafka streams
+        run: mvn clean install -Pprod -Psql -Pinfinispan -Pstreams -Pasyncmem -DskipTests -pl !tests
```

Comment:
Maybe create a separated PR for the Kafka changes?

Reply:
Yeah you're right - I should have removed the kafka storage in a separate PR... I'll do that.